docs/extensibility/form-widgets.md

```diff
@@ -53,6 +53,41 @@ CodeEditor widgets render a versatile code editor that can be used to edit any v
 }
 ```
 
+### Resource
+
+Resource widgets render a dropdown list of specified resources and stores the selected one as a string containing it's name.
```

Comment:
```suggestion
Resource widgets render a dropdown list of specified resources and store the selected one as a string containing its name.
```
- `stores` -> `store`: because `resource widgets` is plural
- `it's` = it is; `its` = belonging to **it**